pkg/execution/driver/httpdriver/parse.go

```diff
@@ -141,13 +141,11 @@ func parseResponse(byt []byte) any {
 		return nil
 	}
 
-	if byt[0] == '{' {
-		// Is the response valid JSON?  If so, ensure that we don't re-marshal the
-		// JSON string.
-		respjson := map[string]interface{}{}
-		if err := json.Unmarshal(byt, &respjson); err == nil {
-			return respjson
-		}
+	// Is the response valid JSON?  If so, ensure that we don't re-marshal the
+	// JSON string.
+	var respjson json.RawMessage
+	if err := json.Unmarshal(byt, &respjson); err == nil {
```

Comment:
Can we use https://pkg.go.dev/github.com/valyala/fastjson?utm_source=godoc#Validate here?